app/models/application_choice.rb

```diff
@@ -1,5 +1,8 @@
 # Candidates can apply to multiple courses via a single Application Form.
 class ApplicationChoice < ApplicationRecord
   belongs_to :application_form, touch: true
-  enum status: { application_complete: 0, conditional_offer: 1 }
+  enum status: {
+    application_complete: 'application_complete',
+    conditional_offer: 'conditional_offer',
+  }
```

Comment:
The real advantage of `enum` is the possibility of using the numbers in place of string.
Maybe I am missing something, what is the point of using enum here? 🤔

Reply:
An enum is a bit smarter than a regular string data type, because it allows you to enumerate all the possible values (`.statuses`), and allows more straightforward validation to ensure a value is within the allowable set... those are the ones I'm aware of!